src/components/index.module.scss

```diff
@@ -70,3 +70,45 @@
   right: 110px;
   z-index: $z_saving;
 }
+
+.printButton {
+  display: flex;
+  align-items: center;
+  position: fixed;
+  right: -62px;
+  top: 240px;
```

Comment:
The `top` here should be something around 416px to appear below the Revision button